backend/src/main/java/fk/prof/backend/http/ApiPathConstants.java

```diff
@@ -1,17 +1,33 @@
 package fk.prof.backend.http;
 
 public final class ApiPathConstants {
-  private ApiPathConstants() {
-  }
+  public static final String DELIMITER = "/";
 
   public static final String AGGREGATOR_POST_PROFILE = "/profile";
 
   public static final String LEADER_POST_LOAD = "/leader/load";
   public static final String LEADER_GET_WORK = "/leader/work";
   public static final String LEADER_POST_ASSOCIATION = "/leader/association";
-  public static final String LEADER_POST_POLICY = "/leader/policy";
 
   public static final String BACKEND_POST_ASSOCIATION = "/association";
   public static final String BACKEND_POST_POLL = "/poll";
   public static final String BACKEND_HEALTHCHECK = "/health";
+
+  public static final String BACKEND_POLICIES = "/policies/";
+  public static final String BACKEND_GET_POLICIES_GIVEN_APPID = "/policies/:appId";
```

Comment:
For now, lets not add API for app or (app, cluster). Just like profile discovery, policy discovery will also require (app, cluster, proc)